custom_model_runner/datarobot_drum/drum/model_adapter.py

```diff
@@ -117,6 +117,9 @@ def load_model_from_artifact(self):
         if (
             self._target_type != TargetType.UNSTRUCTURED
             and not self._custom_hooks[CustomHooks.SCORE]
+        ) or (
+            self._target_type == TargetType.TRANSFORM
+            and not self._custom_hooks[CustomHooks.TRANSFORM]
```

Comment:
So if target type is transform, but there is no hook, we expect artifact to do the transformation, right?
Then we should validate output, am I right?

Reply:
actually now that I think of it, for now this probably isn't appropriate- down the road, we don't want to require that hook for all cases, but as of now it is necessary for transform type. will change to error for now if no hook is provided